service-discovery/README.adoc

```diff
@@ -464,3 +474,214 @@ When service binding is complete, OpenShift redeploys the Quarkus application. W
 . To verify that the Quarkus application is using the Kafka topic, reopen the *Last price* web page that you opened earlier in this tutorial.
 +
 On the *Last price* web page, observe that the price value is continuously updated. The updates show that the Quarkus application is now using the `prices` topic in your Kafka instance to produce and consume messages that correspond to price updates.
+
+[id="con-binding-nodejs-application-using-web-console_{context}"]
```

Comment:
I'm wondering if this is correct?  When I follow the link in the rendered version it goes to the Quarkus example?

Reply:
@mhdawson Ah, yes, now I see what you were experiencing. You're right - this is because the Node.js tutorial is not yet published. I don't think there's anything more to worry about with this - this link will resolve OK once everything is published.